### PR TITLE
Game titles

### PR DESCRIPTION
**Pull request by tfettig22**
## What does this PR do?
- [] New feature (adds functionality)
- [x] Debugging/Refactor
 
### Where should your reviewer start?
- MyGames and SearchGames, and new SquadMemberScreen
### What (if any) features are you implementing?
- All images now have game titles 
### What (if anything) did you refactor?
-  n/a
### What issues does this PR close?
- #49 
### Is there anything that you need from your teammates?
- Check out images, make sure all are appropriately sized/positioned
### Any other comments, questions, or concerns?
-
### How Has This Been Tested?
- [x] I tested my changes in the browser
- [] I tested with Cypress

## Checklist:
- [x] My code follows the style guidelines of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [x] My changes generate no new bugs
- [] I have added tests that prove my fix is effective or that my feature works
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by aspitz1 — APPROVED** [review]
Hey, this works great! Thank you 👍

**Comment by aspitz1** [comment]
I am realizing this was just the game titles, sorry about that. The game titles and the modals are working well from what I can tell. We are still having the repeat key on the child element that I can't find but otherwise, great ;)

File: src/components/MyGames.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ const MyGames = ({ userGames, addGame, removeGame }) => {
                     bottom: 0,
                   }}
                 ></Image>
+                <Text style={styles.gameTitle}>{itemData.item.game_title}</Text>
```
Comment (NickLiggett) [inline src/components/MyGames.js:79]: This makes our lack of cover images way more bearable lol. Awesome idea, Tom.

**Review by NickLiggett — APPROVED** [review]
Great work Tom!

**Pull request merged**